### PR TITLE
Fix target=_blank issues and scss

### PR DESCRIPTION
**Pull request by jalezi**
Fix: #41 and #52

### CODE REVIEW

**Comment by miaerbus** [comment]
Looks good to me. Could you please also fix the share url link that was lost with refactoring? I can create an issue for it if you want.

**Comment by miaerbus** [comment]
Regarding #52, should we add the same useEffect hook to FAQ page as well?

**Comment by jalezi** [comment]
> Regarding #52, should we add the same useEffect hook to FAQ page as well?
I will add.

**Review by miaerbus — APPROVED** [review]

**Comment by jalezi** [comment]
> > Regarding #52, should we add the same useEffect hook to FAQ page as well?
> > I will add.

Done.

**Comment by jalezi** [comment]
Do not merge

**Comment by jalezi** [comment]
@miaerbus please review.

**Comment by miaerbus** [comment]
Now we have two tooltips: one is native from the attribute title, the other is from the Tooltip MUI component.

We could remove the Title component and then it works just fine. Or we remove the native tooltip and update the callback `handleCopy`, so that the title in the Tooltip component is set programmatically.

**Comment by jalezi** [comment]
I would go with Tooltip component.

File: src/components/DoctorCard/PageInfo.js [hunk 1]
```diff
@@ -57,7 +57,11 @@ const PageInfo = ({ doctor }) => {
             <Typography component="div" variant="body1">
               <Icons.Icon name="Phone" />
             </Typography>
-            <Shared.ConditionalLink to={doctor?.phone && `tel:${doctor.phone}`} variant="body1">
+            <Shared.ConditionalLink
+              to={doctor?.phone && `tel:${doctor.phone}`}
+              self
+              variant="body1"
+            >
```
Comment (mihaerzen) [inline src/components/DoctorCard/PageInfo.js:64]: If `doctor?.phone` is `undefined` or falsy, what will happen here? I would propose the following change (conditionally render the component alltogether):

```suggestion
            {doctor?.phone && (<Shared.ConditionalLink
              to={`tel:${doctor.phone}`}
              self
              variant="body1"
            >{doctor.phone}
            </Shared.ConditionalLink>)}
```

File: src/components/DoctorCard/Shared/index.js [hunk 2]
```diff
@@ -11,15 +11,27 @@ import Typography from '@mui/material/Typography';
...
+  );
+};
+export const ConditionalLink = ({ children, to, component = 'div', self, ...props }) => {
+  const link = self ? (
+    <Link href={to} self>
+      {children}
+    </Link>
+  ) : (
+    <Link href={to}>{children}</Link>
+  );
```
Comment (mihaerzen) [inline src/components/DoctorCard/Shared/index.js:31]: I do not think this is needed. Wouldn't this be the same?

```suggestion
  const link = <Link href={to} self={self}>{children}</Link>;
```

File: src/components/DoctorCard/PageInfo.js (hunk 1 above)
Comment (jalezi) [reply src/components/DoctorCard/PageInfo.js:64]: Actually I don't need optional chaining at all. A few lines higher I am alredy condtitionally rendering based on doctor.phone.

File: src/components/DoctorCard/Shared/index.js (hunk 2 above)
Comment (jalezi) [reply src/components/DoctorCard/Shared/index.js:31]: Most likely :)

**Review by miaerbus — APPROVED** [review]
Let's merge this and fix other things in a separate PR

**Review by stefanb — APPROVED** [review]

**Pull request merged**